pkg/docker/index.html

```diff
@@ -140,10 +142,10 @@ <h1 translatable="yes">Can't connect to Docker</h1>
     </div>
     <div class="col-md-12 col-lg-4">
       <div class="panel panel-default" id="containers-storage">
-        <div class="storage-label">Storage space</div>
-        <div class="bar-group">
-          <div class="data"></div>
-          <div class="bar"></div>
+        <div class="panel-body">
+          <div id="containers-storage-details">
+          </div>
+          <a translatable="yes" href="#/storage">configure storage...</a>
```

Comment:
We use `translatable` because that's what `xgettext` expects for `gettext/glade` files, which we abuse to scan our HTML.  But we should definitely work on moving to `translate`.  Thanks for pointing out that there is a official attribute for this.